### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: Cannot find module 'bcrypto.node'

### ISSUE THREAD

**Issue by guillempuche** [ceramicnetwork/js-ceramic#895]
While following your tutorial [How to build a simple notes app with IDX](https://blog.ceramic.network/how-to-build-a-simple-notes-app-with-idx/), I'm facing this error:
```
UnhandledPromiseRejectionWarning: Error: Cannot find module 'bcrypto.node'
```

![imatge](https://user-images.githubusercontent.com/3519924/107119171-16a8ce00-6886-11eb-9752-a370cb7910ae.png)

My packages version:
- @ceramicnetwork/cli: 0.20.8
- @ceramicnetwork/http-client: 0.9.6
- @ceramicstudio/idx-tools: 0.7.0
- uint8arrays: 2.1.2
- key-did-provider-ed25519: 1.0.0

**Comment by oed** [comment]
What version of node are you running @guillempuche ? It should work with node 14.

**Comment by guillempuche** [comment]
Node 14.15.4

**Comment by oed** [comment]
Not exactly sure why this is a problem. Maybe something specific to Windows? 
Can you also try to install `@ceramicnetwork/cli` globally and run `ceramic daemon` to see if you get the same error?

**Comment by guillempuche** [comment]
I think now Ceramic is working...

![cermic working](https://user-images.githubusercontent.com/3519924/107409657-ac558f00-6b0c-11eb-80cd-3fba7c58a21b.jpg)

I think my problem was with node-gyp, see here: https://stackoverflow.com/questions/34546272/cannot-find-module-bcrypt

I was using nvm (Node Version Manager). I used nvm to install Node v14.15.4.

To use bcrypto.node I needed to install globally (according to this https://github.com/kelektiv/node.bcrypt.js/wiki/Installation-Instructions#microsoft-windows and this https://github.com/Microsoft/nodejs-guidelines/blob/master/windows-environment.md#compiling-native-addon-modules) node-gyp and windows-build-tools (Microsoft's windows-build-tools, installed globally as Administrator).

Then, according to this kelektiv/node.bcrypt.js#728 I run npm rebuild in the project where I was using Ceramic.

Thanks, @oed, for your quickness.

If I have a problem, I'll tell you.
